contrib/admintools/copyright.c

```diff
@@ -53,7 +53,7 @@ static void ProcessFile(char *filename,FILE *output, char *dir) {
     time_t now;
     struct tm *tm;
 
-    src = fopen(filename,"r");
+    src = fopen(filename,"rs");
```

Comment:
What's with adding this 's' flag to all the fopen commands?

It's not in any [spec](http://pubs.opengroup.org/onlinepubs/9699919799/functions/fopen.html) that I can find, and [glibc](https://www.gnu.org/software/libc/manual/html_node/Opening-Streams.html) makes no mention of it.

Reply:
You mentioned problems with "r" and asked if it was ok to go with "rb"
"s" would be for shared.

Reply:
I can understand "rb". But where does "s" come from, and what even supports that flag? As far as I can tell, it's nonstandard, so has no effect.

Reply:
Ok, we take it out.
"s" is pretty old, in an operating system known as DOS, and other variants where it began to make sense to read and share at the same time. "r" by itself meant lock others out, so it wasn't shareable until the file was released.